src/server/zset_family.cc

```diff
@@ -1058,6 +1058,16 @@ struct SetOpArgs {
   bool with_scores = false;
 };
 
+void HandleOpStatus(ConnectionContext* cntx, OpResult<SetOpArgs> op_args_res) {
```

Comment:
Please accept `op_args_res` by const&, i.e. `const OpResult<SetOpArgs>&`

Reply:
Actually, you can just pass the status part to this function, as it never uses `SetOpArgs` because it assumes that it is not set.
So in the call sites you would use `HandleOpStatus(cntx, op_args_res.status())` and here you'll accept `OpStatus` (no need for reference as that's an enum)

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2164/commits/da95fb66a44fbbd1822ac941747410ccb195341e